save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/Main.kt

```diff
@@ -50,10 +55,7 @@ fun main() {
     logType.set(if (config.debug) LogType.ALL else LogType.WARN)
     logDebugCustom("Instantiating save-agent version $SAVE_CLOUD_VERSION with config $config")
 
-    platform.posix.chmod(
-        "save-$SAVE_CORE_VERSION-linuxX64.kexe",
-        (S_IRUSR or S_IWUSR or S_IXUSR or S_IRGRP or S_IROTH).toUInt()
-    )
+    "save-$SAVE_CORE_VERSION-linuxX64.kexe".toPath().markAsExecutable()
```

Comment:
`SAVE_CLI_EXECUTABLE_NAME` from Utils?

Reply:
It's in utils in orchestrator, and in agent there's a single place where it is used. When cli is executed, agent receives a complete command from orchestrator.